src/unittest/unittestScripts.ts

```diff
@@ -12,6 +12,22 @@ import base64
 import json
 import traceback
 
+from pathlib import Path
+start_dir = sys.argv[2]
```

Comment:
This is already defined below, can you please reuse this?

Reply:
use, it is possible to reuse it.